quickstart.md

```diff
@@ -63,7 +76,8 @@ This command will create docker containers for each Zendro component:
 * [Keycloak](https://github.com/Zendro-dev/Zendro-dev.github.io/blob/documentation-vb/oauth.md): manage users and roles
 * [Single Page App (SPA)](https://github.com/Zendro-dev/single-page-app): graphical interface to send CRUD requests to a Zendro GraphQL endpoint
 * [API](https://github.com/Zendro-dev/graphql-server): CRUD API that can be accessed through a GraphQL query language
-* [API with authenthication](https://github.com/Zendro-dev/graphiql-auth): An implementation of the GraphQL IDE with Zendro login
+* [API web-interface](https://github.com/Zendro-dev/graphiql-auth): An implementation of the GraphQL IDE with Zendro login and advanced filter functionalities.
+* [traefik reverse-proxy](): A reverse-proxy using traefik that maps the above docker services.
```

Comment:
[traefik reverse-proxy]()  the link is empty

Reply:
_**Done**_ GraphiQL interface
@coeit are you writing another doc for traefik or do we link to traefik official page?